research-software-directory/RSD-as-a-service
# 938 verified organisation

# Verified organisation badge and claim organisation button

Closes #938

Changes proposed in this pull request:
* Verified "banner" is changed into verified badge (icon with the tooltip). See example 1
* If the organisation is not verified we show "claim organisation" button instead. See example 2
* Claim an organisation modal is similar to Register organisation modal on the homepage. There are few minor adaptations:
    * Title is different
    * Start of the email body (content) is different

How to test:
* `make start` to build and generate test data
* Validate register organisation button on the homepage is working properly
* Navigate to organisation overview. Validate verified organisation icond and mouseover message
* Go to next page until you see not verified organisations. Then use icon button. The modal should appear. The title of the modal will be different and the organisation name is already present. 
* Validate Cancel button of the modal
* Validate Send email button working and the content of the email. Please doble check all new messages.

## Example 1 - Verified organisation

![image](https://github.com/research-software-directory/RSD-as-a-service/assets/9204081/7cb059dc-156a-44f2-a082-2ef88680310a)

## Example 2 - Claim organisation button
![image](https://github.com/research-software-directory/RSD-as-a-service/assets/9204081/35c42a5b-932d-4362-bf82-7b6d9c651d5f)

## Example 3 - Claim an organisation form

![image](https://github.com/research-software-directory/RSD-as-a-service/assets/9204081/104598ac-7873-49a5-b5b2-67033b70cdf7)


PR Checklist:
